sdk/python/kfp/_client.py

```diff
@@ -52,21 +55,23 @@ def __init__(self, host=None, client_id=None, namespace='kubeflow'):
     """
 
     self._host = host
```

Comment:
probably change the name to _api_host

Reply:
// I think `host` is can be misleading here as people think they do not need the `https://` schema. `endpoint` or `uri` might be more appropriate.

Reply:
@hongye-sun it seems like `self._host` is not used anymore so maybe we should only keep one, once we clarify the use of API vs UI (above)

Reply:
I think we need two one for API and one for UI. API endpoint can be passed as parameter or env var. UI endpoint can only be customized via env var. The default value of UI endpoint if env var is missing is same as API host. This is useful if the endpoint is from ambassador use case.

Reply:
Oh, sorry. API endpoint, might not need env var support as we support namespace customization.